curriculum/challenges/english/blocks/workshop-cafe-menu/5f33294a6af5e9188dbdb8f3.md

```diff
@@ -1,12 +1,14 @@
 ---
 id: 5f33294a6af5e9188dbdb8f3
-title: Step 5
+title: Step 1
 challengeType: 0
-dashedName: step-5
+dashedName: step-1
 ---
 
 # --description--
 
+In this workshop, you will learn the basics of CSS (Cascading Style Sheets) by building a cafe menu.
```

Comment:
```suggestion
In this workshop, you will practice the basics of CSS (Cascading Style Sheets) by building a cafe menu.
```

this is a reminder from the previous version, as there are the lessons that teach we tend to not use `learn` for workshops